client/js/common/class-object.js

```diff
@@ -1,8 +1,7 @@
 // License: LGPL-3.0-or-later
-const R = require('ramda')
-
-module.exports = (classes='') => R.reduce(
-  (a, b) => {a[b] = true; return a}
-  , {}
-  , R.drop(1, classes.split('.')))
+module.exports = (classes = '') =>
+  classes.split('.').slice(1).reduce((a, b) => {
+    a[b] = true;
+    return a;
+  }, {});
```

Comment:
that sounds right to me, yeah. and I guess it's dropping / splitting out the first one because it's a string with leading `.`s? Like '.name.another-name'?

I dug into this a little more, and `class-object` is only being used by `number-input.js`, `text-input.js`, and `textarea.js`. Of those, only `text-input` is used anywhere else, and it's never passed a `classes` string. So I don't think this (or `number-input` or `textarea`) are used. Should I delete them instead of updating?

Reply:
Ya, I'd delete. Please do in a different PR.

Reply:
Sounds good, will do

Reply:
https://github.com/CommitChange/houdini/pull/966